bschweiz/kennel
# Bs ch8 nov24

# Description
completes ch8 so now you can create and add employees!
Fixes # (employees/create Route)
## Type of change
Please delete options that are not relevant.
- [] Bug fix (non-breaking change which fixes an issue)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
# Testing Instructions
click on employees on the navBar, then Add Empoylee, fill out the fields  and see new employee added successfully
# Checklist:
- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
